README.md

```diff
@@ -25,13 +25,15 @@ socket activation is assumed.
 
 ### Required
 
- * `-command string`: the command to retrieve the key encryption key
+ * `--command string`: the command to retrieve the key encryption key
```

Comment:
```
$ ./kms -help
Usage of ./kms:
  -command string
    	the command to retrieve the key encryption key
  -endpoint string
    	the listen address (ex. unix:///tmp/kms.sock)
  -timeout duration
    	maximum time to cache KEK locally (default 1h0m0s)
```

Can you clarify the difference between single and double dash in this case?

Reply:
There is none (both do the same exact thing).  I just really prefer `--` (and it matches how `kubectl`, `oc`, etc print their flags).

Reply:
I can see if there is a simple way to override how the flags print so they have double dash.

Reply:
#20 to track